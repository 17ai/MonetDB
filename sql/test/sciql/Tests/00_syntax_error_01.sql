-- arrays must have at least one dimension
CREATE ARRAY test_00_01          (payload FLOAT);

-- the [size] shortcut may not be negative
CREATE ARRAY test_00_01_tinyint  (x TINYINT  DIMENSION[-4]);
CREATE ARRAY test_00_01_smallint (x SMALLINT DIMENSION[-4]);
CREATE ARRAY test_00_01_int      (x INTEGER  DIMENSION[-4]);
CREATE ARRAY test_00_01_bigint   (x BIGINT   DIMENSION[-4]);

