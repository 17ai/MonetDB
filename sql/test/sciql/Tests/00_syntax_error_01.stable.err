stderr of test '00_syntax_error_01` in directory 'sql/test/sciql` itself:


# 13:37:16 >  
# 13:37:16 >   mserver5  --debug=10 --set gdk_nr_threads=0  --set "gdk_dbfarm=/export/scratch0/zhang/monet-install/sciql/debug/var/MonetDB" --set mapi_open=true --set mapi_port=38847 --set monet_prompt= --trace --forcemito --set mal_listing=2  "--dbname=mTests_test_sciql" --set mal_listing=0 ; echo ; echo Over..
# 13:37:16 >  

# builtin opt 	gdk_dbname = demo
# builtin opt 	gdk_dbfarm = /export/scratch0/zhang/monet-install/sciql/debug/var/monetdb5/dbfarm
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_alloc_map = no
# builtin opt 	gdk_vmtrim = yes
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	gdk_dbfarm = /export/scratch0/zhang/monet-install/sciql/debug/var/MonetDB
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 38847
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbname = mTests_test_sciql
# cmdline opt 	mal_listing = 0

# 13:37:16 >  
# 13:37:16 >  mclient -lsql -ftest -i -e --host=jarl --port=38847 
# 13:37:16 >  

MAPI  = (monetdb) /var/tmp/mtest-13130/.s.monetdb.39483
QUERY = CREATE ARRAY test_00_01          (payload FLOAT);
ERROR = !CREATE ARRAY: an array must have at least one dimension
MAPI  = (monetdb) /var/tmp/mtest-13130/.s.monetdb.39483
QUERY = CREATE ARRAY test_00_01_tinyint  (x TINYINT  DIMENSION[-4]);
ERROR = !Illegal range of dimension 'x': [0:1:-4]
MAPI  = (monetdb) /var/tmp/mtest-13130/.s.monetdb.39483
QUERY = CREATE ARRAY test_00_01_smallint (x SMALLINT DIMENSION[-4]);
ERROR = !Illegal range of dimension 'x': [0:1:-4]
MAPI  = (monetdb) /var/tmp/mtest-13130/.s.monetdb.39483
QUERY = CREATE ARRAY test_00_01_int      (x INTEGER  DIMENSION[-4]);
ERROR = !Illegal range of dimension 'x': [0:1:-4]
MAPI  = (monetdb) /var/tmp/mtest-13130/.s.monetdb.39483
QUERY = CREATE ARRAY test_00_01_bigint   (x BIGINT   DIMENSION[-4]);
ERROR = !Illegal range of dimension 'x': [0:1:-4]


# 13:37:16 >  
# 13:37:16 >  Done.
# 13:37:16 >  

