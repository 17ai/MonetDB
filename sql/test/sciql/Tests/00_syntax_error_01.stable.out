stdout of test '00_syntax_error_01` in directory 'test/sciql` itself:


# 13:37:16 >  
# 13:37:16 >   mserver5  --debug=10 --set gdk_nr_threads=0  --set "gdk_dbfarm=/export/scratch0/zhang/monet-install/sciql/debug/var/MonetDB" --set mapi_open=true --set mapi_port=38847 --set monet_prompt= --trace --forcemito --set mal_listing=2  "--dbname=mTests_test_sciql" --set mal_listing=0 ; echo ; echo Over..
# 13:37:16 >  

# MonetDB 5 server v11.6.0
# This is an unreleased version
# Serving database 'mTests_test_sciql', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 7.749 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2011 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://jarl.ins.cwi.nl:38847/
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_history.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_compress.sql
# loading sql script: 18_dictionary.sql
# loading sql script: 19_cluster.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 24_zorder.sql
# loading sql script: 25_debug.sql
# loading sql script: 40_geom.sql
# loading sql script: 80_udf.sql
# loading sql script: 99_system.sql

# 16:28:17 >  
# 16:28:17 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=jarl" "--port=31491"
# 16:28:17 >  


# 16:28:17 >  
# 16:28:17 >  "Done."
# 16:28:17 >  

